### DISCUSSION TITLE
2x15 pin right female headers

### DISCUSSION THREAD

**Discussion by kd7eir** [gusmanb/logicanalyzer#105] [Q&A] [answered]
It appears that these are virtually unknown. I can find the male headers with no problem, but the female headers can only be found in 13 or 20 pin, skipping 15 entirely.

I've searched Mouser, Digikey, Aliexpress, and eBay with no luck.

Where has anyone else found these?

**Comment by cefiar** [comment, marked as answer]
I've found these on Digikey which I THINK are correct, but I've yet to order any, or build an analyzer yet (was actually trying to figure out a BOM of sorts).

https://www.digikey.com.au/short/hm22847j

Digikey code:	S5568-ND

I also looked on Mouser, but could not find these type of sockets there.

**Reply by kd7eir** [reply]
Thank you! Your search Fu is far better than mine!

**Reply by cefiar** [reply]
More persistence than search Fu. ;)

Have raised an issue about adding board thicknesses and a BOM to the wiki (which I'm happy to do the grunt work on), which should hopefully make things like this a bit easier.

**Comment by gusmanb** [comment]
I usually buy these from AliExpress: https://es.aliexpress.com/item/1005005288629414.html

I will check this weekend the BOM issue.

Cheers.

**Comment by cefiar** [comment]
Re: BOM

This is what I've come up with. If this seems correct, feel free to grab bits of it/adapt as appropriate. The tables are all done in Github Markdown.

## BOM

### Logic Analyzer

| Qty | Item Detail | Example supplier code |
| --- | --- | --- |
| 1 | Raspberry Pi Pico (or Pico W) | Raspberry Pi SC0915 (Pico) or SC0918 (Pico W) |
| 2 | 20 way straight pin headers 2.54mm (0.100") pitch | Adam Tech PH1-20-UA |
| 1 | 1N4148 General Purpose Diode, SOD80/Mini-MELF package | Onsemi LL4148 |
| 1 | 100K Resistor, 1%, 1/10W, SMD 0603 package | Vishay Dale CRCW0603100KFKEA |
| 1 | 30 pin dual row right angle header socket (2x15 pin) | Sullins Connector Solutions PPPC152LJBN-RC |

If you want to daisy-chain boards, also add the following:

| Qty | Item Detail | Example supplier code |
| --- | --- | --- |
| 1 | 3 way right angle header socket (1x3) | Sullins Connector Solutions PPPC031LGBN-RC |
| 1 | 3 way right angle header pins (1x3) | Adam Tech PH1RA-03-UA |

### Level Shifter

| Qty | Item Detail | Example supplier code |
| --- | --- | --- |
| 1 | 30 pin dual row right angle header socket (2x15 pin) | Sullins Connector Solutions PPPC152LJBN-RC |
| 1 | 30 pin dual row right angle header pins (2x15 pin) | Adam Tech PH2RA-30-UA |
| 12 | 0.1uF (100nF) Ceramic Capacitor 16V 10% SMD 0603 package | YAGEO CC0603KRX7R7BB104 |
| 6 | Texas Instruments TXU0104 IC Translator Unidirectional IC 14-TSSOP package | Texas Instruments TXU0104PWR |

> [!NOTE]
> 
> 20 way headers are not necessary if you buy a Pico H/Pico WH, as the H suffix means they have the headers already soldered on. However, the cost difference may be more than just buying the headers would be and soldering them on yourself. You can also buy the headers in longer break-away lengths that may be cheaper (eg: a single 40 pin length snapped in half, etc).
